book/website/reproducible-research/open/open-hardware.md

```diff
@@ -1,111 +1,359 @@
+Different OSH projects have included different partners at varying stages of their developement.
+On top of user and contributor roles that OSH have in common with open source software, local or global hardware manufacturers may become important partners of your project.
+You may also think early about the people who will eventually have to maintain and repair the hardware.
+To make it easier for them, it also helps to make your hardware designs modular (splitting your hardware in modules which may have alternative designs).
+Anoter specificity of hardware may be the importance of the creation of replication tutorials, workshops, seminars, or training material, which can impact the adoption of hardware designs.
+This is particularly relevant if the OSH is meant to be produced in Do-It-Yourself environments or as a teaching opportunity.
+
+```{note}
+Engineers is still a world full of tech-bros, who got irritated when one does not understand their jargon, and are not very welcoming with beginners and women. 
+
+It is therefore particularly important to Valuing Diversity and Differences in OSH {ref}`cl-new-community-differences`
+
+```
+
+Importantly, it is important to decide whether, when and where you want to engage with, or build one community.
+Most OSH communities are local in comparison to open source software project.
+You may not have the time or skills to build a community, and your project may not need a community to flourish.
+Always be honest with your collaborators and yourself about what support they can expect.
+The [GOSH forum](https://forum.openhardware.science) has been an enabler for finding collaborators for OSH and OSH-related projects.
+
+
+
+
+
+```{admonition} Learning from community roadblocks
+- It is better to keep an open attitude towards diversification and unstable features, and this does not need to be in conflict with the main project being conservative.
+- There should always be space for "extensions" and "plugins" to be seeded and grow in the same community as the main project. 3rd-party plugins should be labeled as "external and unsupported by maniline development" but still encouraged.
+- It is good practice to build modular and extensible hardware, with documented interfaces, to be able to grow a developer community.
+- It is a bad idea to respond to a request for help with the following: "It is 'open source', if you want that to happen, you are free to work on it in your own time". Be honest of what you can do, but do not be a jerk. 
+- If the upstream developers do not have time to write enough documentation, or do not have time to review pull requests, this fact should be stated to avoid generating false expectations.
+- Be ready to delegate (or create new spaces) if you are overwhelmed by a fast-growing community. Otherwise, small projects will stall, the community will be frustrated, and then become fragmented. 
```

Comment:
```suggestion
- Be ready to delegate (or create new spaces) if you are overwhelmed by a fast-growing community. 
Otherwise, small projects will stall, the community will be frustrated, and then become fragmented. 
```